cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -210,8 +274,9 @@ void ImageFlip::imageCallback(const sensor_msgs::ImageConstPtr& color_image_msg)
 	if (rotation_angle==0. || rotation_angle==360. || rotation_angle==-360.)
 	{
 		color_image_turned = color_image;
-		rot_mat.at<double>(0,0) = 1.;
```

Comment:
Just leave it the old way 
`rot_mat.at<double>(0,0) = 1.;`
in this case, because the rotation matrix is always the same format CV_64FC1.